### DISCUSSION TITLE
How can I do the shadow for react native windows?

### DISCUSSION THREAD

**Discussion by taduyde** [microsoft/react-native-windows#10388] [Q&A] [answered]
The shadown for the iOS or android it work as well as screenshot below, but for the windows it didn't work.
![image](https://user-images.githubusercontent.com/8236997/183807824-8367ed54-6a38-46cc-a751-5d6cc3a1265d.png)

**Comment by chrisglein** [comment, marked as answer]
Depends on what API you're using. But many of the shadow APIs are not implemented on Windows: https://github.com/microsoft/react-native-windows/issues/2800
